06-regression.Rmd

```diff
@@ -755,25 +841,90 @@ get_regression_table(lifeExp_model) %>%
 
-1. `intercept = 54.8` corresponds to the mean life expectancy for Africa. This mean life expectancy is treated as a baseline for comparison for the other continents.
-1. `continentAmericas = 18.8` is the difference in mean life expectancies of the
-Americas minus Africa. Note that $18.80 = 73.6 - 54.8$ is the 2nd "mean vs Africa"
-value in Table \@ref(tab:continent-mean-life-expectancies).
-1. `continentAmericas = 15.9` is the difference in mean life expectancy of Asia 
-minus Africa. Note that $15.9 = 70.7 - 54.8$ is the 2nd "mean vs Africa"
-value in Table \@ref(tab:continent-mean-life-expectancies).
-1. `continentEurope = 22.8` is the difference in mean life expectancy of Europe 
-minus Africa. Note that $22.8 = 77.6 - 54.8$ is the 3rd "mean vs Africa"
-value in Table \@ref(tab:continent-mean-life-expectancies).
-1. `continentOceania = 25.9` is the difference in mean life expectancy of Oceania
-minus Africa. Note that $25.9 = 80.7 - 54.8$ is the 3rd "mean vs Africa"
-value in Table \@ref(tab:continent-mean-life-expectancies).
+Just as before, we have the `term` and `estimates` columns of interest, but unlike before, we now have 5 rows corresponding to 5 outputs in our table: an intercept like before, but also `continentAmericas`, `continentAsia`, `continentEurope`, and `continentOceania`. What are these values? First, we must describe the equation for fitted value $\widehat{y}$, which is a little more complicated when the $x$ explanatory variable is categorical:
+
+$$
+\begin{align}
+\widehat{\text{life exp}} &= b_0 + b_{\text{Amer}}\cdot\mathbb{1}_{\mbox{Amer}}(x) + b_{\text{Asia}}\cdot\mathbb{1}_{\mbox{Asia}}(x)
++ b_{\text{Euro}}\cdot\mathbb{1}_{\mbox{Euro}}(x) + b_{\text{Ocean}}\cdot\mathbb{1}_{\mbox{Ocean}}(x)\\
+&= 54.8 + 18.8\cdot\mathbb{1}_{\mbox{Amer}}(x) + 15.9\cdot\mathbb{1}_{\mbox{Asia}}(x)
++ 22.8\cdot\mathbb{1}_{\mbox{Euro}}(x) + 25.9\cdot\mathbb{1}_{\mbox{Ocean}}(x)
+\end{align}
+$$ 
+
+Let's break this down. First, $\mathbb{1}_{A}(x)$ is what's known in mathematics as an "indicator function" that takes one of two possible values:
+
+$$
+\mathbb{1}_{A}(x) = \left\{
+\begin{array}{ll}
+1 & \text{if } x \text{ is in } A \\
+0 & \text{if } \text{otherwise} \end{array}
+\right.
+$$
+
+In a statistical modeling context this is also known as a "dummy variable". In our case, let's consider the first such indicator variable:
+
+$$
+\mathbb{1}_{\mbox{Amer}}(x) = \left\{
+\begin{array}{ll}
+1 & \text{if } \text{country } x \text{ is in the Americas} \\
+0 & \text{otherwise}\end{array}
+\right.
+$$
+
+Now let's intepret the terms in the estimate column of the regression table. First $b_0 =$ `intercept = 54.8` corresponds to the mean life expectancy for countries in Africa, since for country $x$ in Africa we have the following equation:
```

Comment:
"interpret" instead of "intepret." I'll do a spellcheck here to make sure I'm not missing others.